### ISSUE TITLE
Snapp Website Down?

### ISSUE THREAD

**Issue by MultiMarcs** [jmoenig/Snap#1285]
Hello, I recently went into the Snapp! website, but when I tried converting the project to an exe it gave me an error saying, "Whoops, server is down" Is this a bug or an issue?

**Comment by jmoenig** [comment]
roping @bromagosa into this discussion...

**Comment by bromagosa** [comment]
And roping @rydion also :P

(I guess we should add a hyperlink to Snapp!'s GitHub repo in the website?)

**Comment by MultiMarcs** [comment]
perhaps a visual image would help 
![image](https://cloud.githubusercontent.com/assets/20291854/16589962/2a0f6d5c-429b-11e6-8cf1-e145da20abe4.png)

**Comment by Rydion** [comment]
Hi, well either for some reason you truly couldn't connect to the server or there was an error and the message is kinda misleading. If you could share the project you were using I'll try to find out what happened.

**Comment by MultiMarcs** [comment]
https://drive.google.com/file/d/0B9UVbDjJzaSkbTU2WUd1LXd4eVk/view?usp=sharing

since the file is to big to post it here in github, I decided to put it up in my drive, I also tried compiling another project but I still recieved the same error :/ anyways hope theres something that can be done :)

**Comment by MultiMarcs** [comment]
Bump?

**Comment by Rydion** [comment]
Hi,
sorry for the delay but I had to take care of some things this past week.

Anyways, I tested the project with my local setup and after finding and correcting a small bug everything worked well (I compared the execution flow with the official snap! and everything seemed fine).

![5 nights](https://cloud.githubusercontent.com/assets/15717238/16816790/fa05c3a8-493f-11e6-985d-bb85a3c06b6b.png)

But when I try to generate the project using the (updated) version hosted on http://snapp.citilab.eu/ I get the same message as you.

So basically it's some kind of problem with the server, either the installation there is faulty or it just can't handle such a big file and it times out.

I will investigate the issue further, in the meanwhile I could upload a windows executable of the project to google drive if you want/need it.
